docs/projects/amt/adrs/0008-systemcard-storage.md

```diff
@@ -0,0 +1,50 @@
+# AMT-0008 System card Storage
+
+## Context
+
+By default, Kubernetes pods use ephemeral storage, which is tied to the pod's lifecycle.
+When the pod terminates or restarts, all data is lost. The `/tmp/` directory, being part of
+the system's temporary file storage, is cleared during reboots or pod restarts, resulting in
+the deletion of system_cards. Therefore, we need a different kind of storage to preserve the data.
+
+## Assumptions
+
+* The system card data is small to moderate in size and fits within the capabilities of Postgres' JSONB storage.
```

Comment:
Optionally consider making this more explicit: what is the average and maximum file size we expect?